scripts/check_style.sh

```diff
@@ -20,6 +20,13 @@ EXCLUDE_FILES=(
 EXCLUDE_FILES_JOINED=$(printf "%s\|" "${EXCLUDE_FILES[@]}")
 EXCLUDE_FILES_JOINED=${EXCLUDE_FILES_JOINED%??}
 
+INCLUDE_DIRECTORIES=(
+  "libevmasm"
+  "libsolidity/ast"
+)
+INCLUDE_DIRECTORIES_JOINED=$(printf "%s/* " "${INCLUDE_DIRECTORIES[@]}")
+INCLUDE_DIRECTORIES_JOINED=${INCLUDE_DIRECTORIES_JOINED%?}
```

Comment:
This seems unnecessary to me. Why this reassignment is required? Is it to remove the trailing space? I guess this will not be a problem for the grep anyway.

Reply:
You're right, that is indeed superfluous.